mappings/net/minecraft/server/command/DataCommand.mapping

```diff
@@ -1,4 +1,12 @@
 CLASS ub net/minecraft/server/command/DataCommand
+	CLASS ub$a Modify
```

Comment:
I don't think class names should be verbs...

Reply:
Hmm, it is a functional interface that is initialized with a function for either `append`, `prepend`, `set` or `merge` and called by `executeModify` to execute the respective function.
`ModifyOperation` sounds good to me.